docs/design.md

```diff
@@ -431,3 +432,106 @@ that don't use PMEM-CSI at all.
 Users must take care to create PVCs first, then the pods if they want
 to use the webhook. In practice, that is often already done because it
 is more natural, so it is not a big limitation.
+
+### Operator API
+
+PMEM-CSI operator facilitates deploying and managing the [PMEM-CSI driver](https://github.com/intel/pmem-csi) on a Kubernetes cluster. 
+This operator is based on the CoreOS [operator-sdk](https://github.com/operator-framework/operator-sdk) tools and APIs.
+This section describes the custom resource definition API provided for deploying the PMEM-CSI driver.
+
+#### Deployment CRD API
+
+`Deployment` is a cluster-scoped Kubernetes resource in the
+`pmem-csi.intel.com` API group. It describes how a PMEM-CSI driver
+instance is to be created.
+
+The operator will create objects in the namespace in which the
+operator itself runs unless the object type is not namespaced.
```

Comment:
this sentence is not introduced by current change, but caught my eye, "unless + not" results double negation which is more difficult to read, while logically correct. Can this double negation be expressed with simpler no-negation sentence?
We have "will create if not object is not namespaced" so is it equal to "will create if object is namespaced" ?

Reply:
Yes, that's better.

Reply:
Fixed the wording.